packages/graphql/src/schema/make-schema-to-augment.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { DefinitionNode, DocumentNode, ObjectTypeDefinitionNode } from "graphql";
+import { Kind } from "graphql";
+import getFieldTypeMeta from "./get-field-type-meta";
+
+function parseJwtPayload(jwtPayloadAnnotatedTypes: DefinitionNode[]) {
+    if (jwtPayloadAnnotatedTypes.length > 1) {
```

Comment:
Not important now, but I believe that in the future we can have these validations as validation rules as it was done with the new custom rule`packages/graphql/src/schema/validation/custom-rules/directive-argument-of-correct-type.ts`.